packages/account/src/Sections/Security/SelfExclusion/self-exclusion.jsx

```diff
@@ -97,44 +104,18 @@ class SelfExclusion extends React.Component {
         const { currency, is_eu, is_cr } = this.props;
         const errors = {};
         // Regex
-        const is_number = /^\d+(\.\d+)?$/;
-        const is_integer = /^\d+$/;
-        const max_number = 9999999999999;
-        const six_weeks = 60480; // in minutes
-
-        // Messages
-        const valid_number_message = localize('Should be a valid number');
-        const more_than_equal_zero_message = localize('Please input number greater than or equal to 0');
+        const max_number = this.exclusion_fields_settings.max_number;
+        const six_weeks = this.exclusion_fields_settings.six_weeks; // in minutes
+
         const more_than_zero_message = localize('Please input number greater than 0');
-        const max_number_message = localize('Reached maximum number of digits');
-        const max_decimal_message = (
-            <Localize
-                i18n_default_text='Reached maximum number of decimals: {{decimal}}'
-                values={{ decimal: getDecimalPlaces(currency) }}
-            />
-        );
 
-        const getLimitNumberMessage = current_value => (
-            <Localize
-                i18n_default_text='Please enter a number between 0 and {{current_value}}'
-                values={{ current_value }}
-            />
-        );
+        const getSmallestMinValue = decimals =>
+            `0.${Array(decimals - 1)
+                .fill(0)
+                .join('')}1`;
```

Comment:
can we consider zero decimal places as well?
For `IDK` currency we don't allow any decimal places. We haven't added `IDK` currency yet but it's nice to consider to zero decimal place.
More info - https://trello.com/c/Hr8EqpgP/2996-fe-issues-for-multiplier

Reply:
Alright, I added a new condition for zero decimals